.github/workflows/release-drafter.yml

```diff
@@ -8,7 +8,7 @@ on:
   workflow_dispatch:
 
 jobs:
-  stale:
+  release-drafter:
```

Comment:
Github has really go release not generation tool by now. We might just consider migrating to that at some point?

Reply:
Upsie. Not sure what I was trying to type there 🙈 
What I was trying to say is that GitHub as a release-note generation tool integrated since a while now. You can configure it via a `.github/release.yml`. See [here](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes) for more information.